### PR TITLE
Read config from file and initialize datastore with it

### PR DESCRIPTION
**Pull request by tkyi**
Given `filename: 'some/path/to/config.json`, imdb will be initialize with that data.

### CODE REVIEW

**Comment by jer** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -9,29 +9,40 @@
...
 const imdb = new Imdb();
 
 // Will output: null
 console.log(imdb.get('table1', 'key1'));
 
-// Initialization with a database
+
+// Initialization with a filename
 const imdb2 = new Imdb({
+  filename: 'some/path/to/config.json'
```
Comment (nkatzman) [inline README.md]: can you give an example of what this file needs to look like?

**Comment by nkatzman** [comment]
👍

**Comment by FenrirUnbound** [comment]
👍 

l41yznyklpccb0b16

**Comment by jer** [comment]
👍

**Pull request merged**